js/websocket_app/src/stream/Data_Stream.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import useWebSocket, { ReadyState } from 'react-use-websocket';
+
+function Data_Stream() {
```

Comment:
[Suggestion / Naming convention] Naming convention for react components is `AllCapsNoUnderscores`